simexpal/launch/slurm.py

```diff
@@ -125,12 +125,15 @@ def _represents_int(x):
 		sbatch_args = ['sbatch', '-J', experiment.display_name]
 		if self.queue:
 			sbatch_args += ['-p', self.queue]
+		if experiment.is_exclusive:
+			sbatch_args += ['--exclusive']
+		else:
```

Comment:
This therefore (see comment on `exclusive` and `num_threads`) must not be in an else branch. We want to set `num_threads` in any way.